tests/fixtures/tests/certificate-pascal-case-example.php

```diff
@@ -86,11 +89,11 @@ public function update_uses_form_request_validation()
     public function update_behaves_as_expected()
     {
         $certificate = factory(Certificate::class)->create();
-        $certificate = $this->faker->word;
+        $update = factory(Certificate::class)->make();
+
+        $response = $this->put(route('certificate.update', $certificate), $update->toArray());
```

Comment:
Same as above. The combination of `make` and `toArray` is _too fancy_ and _too specific_. I want to ensure the code Blueprint generates communicates well and is approachable for developers to modify.